srcs/image/minimap.c

```diff
@@ -0,0 +1,54 @@
+/* ************************************************************************** */
+/*                                                                            */
+/*                                                        :::      ::::::::   */
+/*   minimap.c                                          :+:      :+:    :+:   */
+/*                                                    +:+ +:+         +:+     */
+/*   By: elouisia <elouisia@student.42.fr>          +#+  +:+       +#+        */
+/*                                                +#+#+#+#+#+   +#+           */
+/*   Created: 2022/09/28 16:42:27 by elouisia          #+#    #+#             */
+/*   Updated: 2022/10/02 15:36:50 by elouisia         ###   ########.fr       */
+/*                                                                            */
+/* ************************************************************************** */
+
+#include "cub3d.h"
+
+void	ft_render_minimap(t_cub_data *data, int colour)
```

Comment:
FYI I entirely modified this function in f03c029

Reply:
Such a busy bee